dist/about.html

```diff
@@ -0,0 +1,190 @@
+                    </div>
+                </div>
+            </nav>
+            <div class="home-div-about">
+                <h4>"Ignite New Ideas"</h4>
+                <h1>StartUp BattleField 2023</h1>
+                <hr class="desktop-hidden">
+                <div class="about-card">
+                    <p>
+                        The original startup conference — stays fresh, relevant and focused on founders, investors
+                        - BattleField delivers the tools, knowledge, and connections to help you make it happen.
+                    </p>
+                </div>
+                <div class="email-about">
+                    <p>Please contact us per Email for any further questions about StartUp BattleField! <span>hackathon2023@startup.org<span></p>
+                </div>
+            </div>
+        </header>
+
+        <section class="logo-about container-fluid">
+            <h3>StartUp BattleField<br>hackathon 2023 logo</h3>
+            <hr>
+            <p>
+                The logo of StartUp BattleField 2023 was decided through the logo competition from <br><span>8. January to 7. January.<span>
+            </p>
+            <img src="/dist/images/startup-battlefield.jpeg" alt="" srcset="">
+        </section>
+
+        <section class="past-about container-fluid">
+            <h3>See the past StartUp Hackathons</h3>
+            <hr>
+            <p>
+                Take a look at the last two StartUp Hackathons which took place in Uganda and in Rwanda.
+            </p>
+            <div class="row g-3">
+                <div class="col col-6 col-md-6 col-sm-12 group1">
+                    <img src="/dist/images/about_2013.png" alt="" srcset="">
+                </div>
+                <div class="col col-6 col-md-6 col-sm-12 group2">
+                    <img class="img2 mb-4" src="/dist/images/about_2011.png" alt="" srcset="">
+                </div>
+            </div>
+        </section>
+
+        <section class="container-fluid sponsors">
+            <h3 class="d-flex justify-content-center">Sponsors</h3>
+            <hr>
+            <div class="row">
+                <div class="col col-4 col-md-3 col-lg-3">
+                    <img src="../dist/images/amazon.png" alt="" srcset="">
+                </div>
+                <div class="col col-4 col-md-2 col-lg-2">
+                    <img src="../dist/images/google2.png" alt="" srcset="">
+                </div>
+                <div class="col col-4 col-md-2 col-lg-2">
+                    <img src="../dist/images/google.png" alt="" srcset="">
+                </div>
+                <div class="col col-6 col-md-2 col-lg-2 apple">
+                    <img src="../dist/images/apple.png" alt="" srcset="">
+                </div>
+                <div class="col col-6 col-md-3 col-lg-3 airbnb">
+                    <img src="../dist/images/airbnb.png" alt="" srcset="">
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Please, on the about page, hide the partner's section for the desktop view and only show it for mobile, as it is in the original design. One method is to add 'display none' for the partner's section in the media query on the desktop.👍️